Bio/SVDSuperimposer/__init__.py

```diff
@@ -14,7 +14,7 @@
 
 
 try:
-    from numpy import dot, transpose, sqrt
```

Comment:
This was not on the list of things we suggested changing. I would be inclined not to touch this since you've left it with inconsistent import styles.